src/main/java/org/schemaspy/util/JDot.java

```diff
@@ -0,0 +1,64 @@
+package org.schemaspy.util;
+
+import org.apache.commons.io.IOUtils;
+
+import javax.script.ScriptContext;
+import javax.script.ScriptEngine;
+import javax.script.ScriptEngineManager;
+import javax.script.ScriptException;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.InputStream;
+
+public class JDot {
+
+    private static final String SIMULATE_VERSION = "2.26";
+    private static final String ICON_SIZE = " , width: \"261px\" , height: \"261px\"";
+    private static final int JAVA_SCRIPT_ENGINE_MEMORY_SIZE = 1 << 30;
```

Comment:
I was thinking more like ONE_GB.

The usage location will tell you what's the variable is used for but a human readable explanation for the value would be ONE_GB.